package.json

```diff
@@ -13,6 +13,7 @@
     "test-ci": "jest --config test/unit/jest.conf.js --testResultsProcessor=./node_modules/jest-junit-reporter",
     "lint": "eslint --ext .js,.vue src test/unit",
     "build": "node build/build.js",
+    "server:prod": "http-server dist -p 8000 -c-1 --cors",
```

Comment:
When I run this script, I get the error below. It looks like you need to add `http-server` as a dependency.

```
phen...body $ npm run server:prod

> phenotypr-body@0.1.0 server:prod /Users/harrelst/Development/pcori/phenotypr-body
> http-server dist -p 8000 -c-1 --cors

sh: http-server: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! phenotypr-body@0.1.0 server:prod: `http-server dist -p 8000 -c-1 --cors`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the phenotypr-body@0.1.0 server:prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Reply:
added need for global dependency "http-server" for production web server